.github/actions/install-apt-packages/action.yml

```diff
@@ -0,0 +1,11 @@
+name: Apt package installer
+jobs:
+description: 'Apt package installer'
+runs:
+  using: "composite"
+  steps:
+    - name: Install packages
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install libmagickwand-dev imagemagick ghostscript pdftk enscript
```

Comment:
I think this list might've been from me throwing things against a wall and seeing what stuck (and the answer might be none of them...). That said, I think we'll likely need this step in some way, so I'm good with keeping it here for now.

Reply:
I think at some point I removed it and it was needed, I don't remember why exactly.